test/integration/dialects/postgres/data-types.test.js

```diff
@@ -245,4 +247,61 @@ if (dialect === 'postgres') {
     });
 
   });
+
+  if (current.dialect.supports.GEOMETRY) {
+    describe('GEOMETRY', () => {
+
+      it('should work with geometry object including crs', function() {
+        const User = this.sequelize.define('User', {
+          username: DataTypes.STRING,
+          geometry: DataTypes.GEOMETRY
+        });
+
+        const point = {
+          type: 'Point', coordinates: [39.807222, -76.984722],
+          crs: {
+            type: 'name',
+            properties: {
+              name: 'EPSG:4326'
+            }
+          }
+        };
+        User.sync({ force: true }).then(() => {
```

Comment:
please add a `return` here so that the test knows when it is done

Reply:
Ah , missed those. Thanks. Done